### ISSUE TITLE
Вынос time.sleep во время вывода текста в отдельную функцию

### ISSUE THREAD

**Issue by aleckonim** [Andrii-Ishchenko-dp/Story#3]
Предлагаю все места, где делается пауза во время вывода текста:

1. Заменить на "ввод Enter -> time.sleep()".
2. Вынести в отдельную функцию и заменить во всех соответствующих местах вызов sleep на вызов данной функции.

Функцию можно назвать, например, `pause`. Она может принимать аргумент, отвечающий за время для sleep.
Также само время можно вынести в константы. Например:
```
DEFAULT_PAUSE = 3  # in seconds
SHORT_PAUSE = 1
LONG_PAUSE = 5
```
При этом аргумент, который принимает функция `pause` может иметь значение по-умолчанию равное `DEFAULT_PAUSE`.

При этом константы выше можно вынести в отдельный модуль `const.py`.

**Comment by JIboJIb** [comment]
Теж хотів запропонувати поки не добрався до цього, там де багато строк не встигаєщ прочитати. Треба зробити привязку до кількості символів напевно

**Comment by Andrii-Ishchenko-dp** [comment]
Да, ребята, спасибо за обратную связь. Всё-таки думаю сделать вывод текста по нажатию клавиши, чтобы не гадать со временем. Хочу также перевести персонажей и их способности в классы для удобства.
